.cirrus.yml

```diff
@@ -1,3 +1,29 @@
+task:
+  auto_cancellation: true
+
+  env:
+      PY_VER: "3.6"
+
+  name: Black Formatting
+  container:
+      image: python:latest
+  env:
+      PATH: $HOME/conda/bin:$PATH
+  conda_script:
+      - curl https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > install.sh
+      - bash install.sh -b -p $HOME/conda
+      - conda update -yn base conda
+      - conda install -y python=$PY_VER
+      - rm install.sh
+
+  install_script:
+    # pip >= 19.0 crashes on Linux with Python 3.7
+    - python -m pip install --retries 3 --upgrade "pip<19.0"
+    - pip install --retries 3 -r requirements.txt
```

Comment:
we really only need `` pip install `cat requirements/development.txt | grep black` ``